### PR TITLE
Fix windows flake

### PR DESCRIPTION
**Pull request by abidlabs**
Fixes the Windows flake (should have included this in the earlier PR) -- cc @freddyaboulton @space-nuko

### CODE REVIEW

File: test/test_blocks.py [hunk 1]
```diff
@@ -1280,7 +1280,10 @@ def greet(s, prog=gr.Progress(track_tqdm=True)):
                 if msg["msg"] == "send_hash":
                     await ws.send(json.dumps({"fn_index": 0, "session_hash": "shdce"}))
                 if msg["msg"] == "progress":
-                    progress_updates.append(msg["progress_data"])
+                    if msg[
+                        "progress_data"
+                    ]:  # Ignore empty lists which sometimes appear on Windows
```
Comment (freddyaboulton) [inline test/test_blocks.py:1285]: Darn windows

**Review by freddyaboulton — APPROVED** [review]
Thanks @abidlabs !

**Comment by abidlabs** [comment]
Thanks @freddyaboulton for the review!

File: test/test_blocks.py (hunk 1 above)
Comment (abidlabs) [reply test/test_blocks.py:1285]: yepp

**Pull request merged**